src/main/java/seedu/duke/atomichabit/atomichabit/AtomicHabit.java

```diff
@@ -0,0 +1,26 @@
+package seedu.duke.atomichabit.atomichabit;
```

Comment:
Hmm would atomichabit.feature be a better package? It's a bit confusing this way